File: hook/hookapp/src/main/java/com/baiiu/hookapp/MyApplication.java
package com.baiiu.hookapp;

import android.app.Application;
import android.content.Context;

/**
 * author: zhuzhe
 * time: 2020-01-15
 * description:
 */
public class MyApplication extends Application {
    private static Context sContext;

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
    }

    @Override
    public void onCreate() {
        super.onCreate();
        sContext = this;
    }

    public static Context getContext() {
        return sContext;
    }
}

File: someApplications/researchStudy/src/main/java/com/baiiu/workhard/MyApplication.kt
package com.baiiu.workhard

import android.app.Application
import android.content.Context

/**
 * author: zhuzhe
 * time: 2020-04-21
 * description:
 */
class MyApplication : Application() {

    companion object {
        lateinit var sContext: Context
    }

    override fun onCreate() {
        super.onCreate()
        sContext = this
    }

}